CHANGES/4700.feature

```diff
@@ -0,0 +1 @@
+AioHTTPTestCase more async friendly
```

Comment:
It'd be great if you could expand what such a friendliness means.

Reply:
Yes, sure